### ISSUE TITLE
Crash when create project

### ISSUE THREAD

**Issue by tung2744** [oursky/create-react-native-skygear#60]
```
/Users/tungw/.config/yarn/global/node_modules/create-react-native-skygear/dist/types.js:10
            throw new Error("You must provide a value");
            ^
Error: You must provide a value
    at Function.Maybe.just (/Users/tungw/.config/yarn/global/node_modules/create-react-native-skygear/dist/types.js:10:19)
    at Maybe.map (/Users/tungw/.config/yarn/global/node_modules/create-react-native-skygear/dist/types.js:27:22)
    at Command.<anonymous> (/Users/tungw/.config/yarn/global/node_modules/create-react-native-skygear/dist/commands/init.js:118:54)
    at Command.listener (/Users/tungw/.config/yarn/global/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:182:13)
    at Command.parseArgs (/Users/tungw/.config/yarn/global/node_modules/commander/index.js:654:12)
    at Command.parse (/Users/tungw/.config/yarn/global/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/Users/tungw/.config/yarn/global/node_modules/create-react-native-skygear/dist/index.js:15:9)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
```

**Comment by rickmak** [comment]
@tung2744 any step to reproduce?

**Comment by cychiuae** [comment]
Fixed in 7b19b1a
